data/coalition missions.txt

```diff
@@ -60,12 +60,19 @@ mission "Coalition: First Contact"
 			label oppressive
 			`	The spider says, "In separate boxes, they each species keep. And in subtle ways, your development and discoveries they limit. And on your technology, restrictions they place."`
 			`	The beetle says, "Or, wrong are we? And the jump drive that brought you here, a gift from the Quarg it is? They its secret no longer hoard?" You tell it that the Quarg do indeed refuse to give jump drives to humans, and it says, "Then, prisoners indeed you are."`
-			label next
+			branch next
+				has "First Contact: Hai: offered"
```

Comment:
No. `"First Contact: Hai: offered"` means that you have met the Hai. So now the player can mention them. Look at the file: with `branch next` (which is when the player _has_ met the Hai), he/she gets the player to mention the Hai. Otherwise (`label next`), the player does not get to mention the Hai. Unless I'm mistaken....